ocs_ci/framework/main.py

```diff
@@ -128,6 +138,8 @@ def process_ocsci_conf(arguments):
     )
 
     args, unknown = parser.parse_known_args(args=arguments)
+    marker = arguments[arguments.index("-m") + 1] if "-m" in arguments else ""
+    load_config(get_tests_config(marker))
```

Comment:
This is very problematic part - as e.g. for upgrade we are using this:
```
-m pre_upgrade or pre_ocs_upgrade or ocs_upgrade or post_ocs_upgrade or post_upgrade
```

For post upgrade execution we are using:
```
-m tier1 or tier_after_upgrade
```

And it can be whatever expression user will use with and and or operators and brackets.

Reply:
This is a grate example to use configuration file.
You can use a different configuration for different markers on the same test.
We also can have a "global" directory, for test which uses the same parameters with different markers.